src/olympia/devhub/tests/test_views_edit.py

```diff
@@ -726,6 +729,34 @@ def test_edit_description_does_not_affect_privacy_policy(self):
         assert addon.description_id
         assert addon.description == u'Sométhing descriptive.'
 
+    def test_no_manage_billing_when_no_subscription(self):
+        response = self.client.get(self.url)
+
+        assert pq(response.content)('.stripe-customer-portal').length == 0
+
+    def test_no_manage_billing_when_subscription_process_not_completed(self):
```

Comment:
The case has no subscription existing. Do you want to have a case where a subscription exists, but it isn't marked as completed? Or will that case never happen?

Reply:
So, it does have one actually. When we create a `PromotedAddon` with a group like `VERIFIED`, a subscription is automatically created.